### ISSUE TITLE
Defined root paths / quality profiles on a per-user basis

### ISSUE THREAD

**Issue by feerlessleadr** [Fallenbagel/jellyseerr#171]
### Description

Currently with Ombi I'm able to set the default root path and quality/language profile on a per user basis. This allows for people who are not technically inclined to not have to worry about selecting a root folder or quality setting (and therefore I don't have to check they did it right every time they make a request). 

### Desired Behavior

Something like Ombi where quality profiles and root paths can be set both globally, but also on a per user basis. 

![image](https://user-images.githubusercontent.com/288895/176767296-b6b8583b-4b0d-46bc-b423-4f2e8627a6f0.png)

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by Fallenbagel** [comment]
This is like the filter system in petio right? It's one of the upcoming features we got planned c:

**Comment by feerlessleadr** [comment]
To be honest I haven't used petio before, so I'm not sure. Just looking through their docs, I'm not 100% sure it is the same thing (but I can't tell what conditions you can set). 

Basically, what I'm looking for is to be able to have jellyseer say if User A requests a movie, then apply radarr quality profile 720p and root folder /media/parentsmovies/, Same thing for TV except with Sonarr. I essentially want all of the requests from User A to be assigned a specific quality profile from sonarr/radarr, as well as specific root folder from sonarr/radarr (which would override the default settings for jellyseer. 

Hopefully that makes sense.

**Comment by Fallenbagel** [comment]
Yeah I know what you mean and in petio it does the same thing. In addition to root folders it can also set stuff like age rating and stuff. That's planned for the future c:

**Comment by feerlessleadr** [comment]
Awesome - thanks for the confirmation. This is the last thing keeping me on ombi before moving my family over to jellyseer.

**Comment by feerlessleadr** [comment]
Just wanted to check in on this to see if this is still on the roadmap? Thanks

**Comment by feerlessleadr** [comment]
Hi @Fallenbagel - Any update on whether this is still on the roadmap?

**Comment by landski** [comment]
Just wanted to bump this one, been waiting 3 years for overseerr to implement this [https://github.com/sct/overseerr/issues/609](url)

Starting to think this is too big a job with the way overseer/jellyseerr is coded.

**Comment by EliantenHolder** [comment]
Setting default quality profiles per user would be tremendously helpful! I have set the default for my users to be max 1080p but for personal viewing I want 4k with all the bells and whistles. So now on every request I have to change the quality profile which is not a huge deal but it would take away my main frustration with sharing my Jellyseerr instance.
